app/graphql/types/integrations/sync_invoice_input.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Types
+  module Integrations
```

Comment:
As I can see, we don't need this class

Reply:
I moved the argument to the input and we're using this input now.